docs/sources/operators-guide/securing/authentication-and-authorization.md

```diff
@@ -7,7 +7,9 @@ weight: 20
 
 # Grafana Mimir authentication and authorization
 
-Grafana Mimir is a multi-tenant system where tenants can query metrics and alerts that include their tenant ID. The query takes the tenant ID from the `X-Scope-OrgID` parameter that exists in the HTTP header of each request.
+Grafana Mimir is a multi-tenant system where tenants can query metrics and alerts that include their tenant ID.  
+The query takes the tenant ID from the `X-Scope-OrgID` parameter that exists in the HTTP header of each request.  
+Queries can also be federated across multiple tenants when you enabled `tenant-federation`, refer to [Configuration Parameters]({{< relref "../reference-configuration-parameters/index.md#configuration-parameters" >}}). The tenant IDs involved need to be specified separated by a `|` character in the 'X-Scope-OrgID' header. 
```

Comment:
Could you add an example of what supplying mutiple tenant IDs looks like? E.g. `tenant-1|tenant-2|tenant-3`